source/guide_eturnal.rst

```diff
@@ -0,0 +1,242 @@
+.. author:: this.ven <https://this.ven.uber.space>
+
+.. tag:: STUN
+.. tag:: TURN
+.. tag:: webRTC
+.. tag:: Erlang
+
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/eturnal.png
+      :align: center
+
+#######
+eturnal
+#######
+
+.. tag_list::
+
+eturnal_ is a modern, straightforward STUN and TURN server written in Erlang. Clients can connect using UDP, TCP, or TLS over IPv4 or IPv6. For authentication, eturnal supports the mechanism described in the `REST API for Access to TURN Services`_ specification.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`firewall ports <basics-ports>
+  * :manual:`cron jobs <daemons-cron>
+
+License
+=======
+
+eturnal is released under the `Apache License, Version 2.0`_.
+
+Prerequisites
+=============
+
+Set up your URL:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Check the `latest release`_ and download a precompiled tarball from https://eturnal.net:
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ wget https://eturnal.net/download/linux/eturnal-1.10.1-linux-x64.tar.gz
+  [isabell@stardust ~]$ tar xzf eturnal-1.10.1-linux-x64.tar.gz
+  [isabell@stardust ~]$
+
+Configuration
+=============
+
+Open Firewall Ports
+-------------------
+eturnal needs 2 listen ports, plus a port range for relaying UDP connections. So lets open 5 ports.
+
+.. include:: includes/open-port.rst
+
+eturnal Config
+--------------
+
+Generate a DH-file and a random super long secret (64 characters).
+
+.. code-block:: console
+
+  [isabell@stardust ~]$ openssl dhparam -out $HOME/eturnal/etc/dh-parameters.pem 4096
+  Generating DH parameters, 4096 bit long safe prime, generator 2
+  This is going to take a long time
+  ........+.................................+.....................+..................
+  ....................+.....................................+........+...............
+  [...]
+  [isabell@stardust ~]$ openssl rand -hex 32
+  [isabell@stardust ~]$ <super-long-secret>
+  [isabell@stardust ~]$
+
+Backup the standard config: ``mv ~/eturnal/etc/eturnal.yml ~/eturnal/etc/eturnal.yml.bkp``.
+Create a new file at ``~/eturnal/etc/eturnal.yml`` and replace values in brackets ``<value>`` with your values.
+
+.. code-block:: yaml
```

Comment:
the code block is broken, you need to indent it, have a look at https://deploy-preview-1300--uberlab.netlify.app/guide_eturnal/